src/client/BuilderClient.ts

```diff
@@ -318,4 +320,79 @@ export class BuilderClient {
 
     return body.data
   }
+
+  public async uploadLandRedirectionFile(
```

Comment:
As it's not uploading something in the client, what do you think about changing the name to:
```suggestion
  public async createLandRedirectionFile(
```